README.md

```diff
@@ -1,76 +1,205 @@
-> A controller that polls Athenz to update the ServiceRole and ServiceRoleBinding Istio custom resources.
-
 # K8s-athenz-istio-auth [![Build Status](https://api.travis-ci.com/yahoo/k8s-athenz-istio-auth.svg?branch=master)](https://travis-ci.com/yahoo/k8s-athenz-istio-auth)
-K8s-athenz-istio-auth is a controller that polls Athenz to update the ServiceRole and ServiceRoleBinding Istio custom 
-resources; it watches all namespaces and looks up the corresponding Athenz domains associated with them. 
-
-### Architecture
-![Screenshot](images/architecture.png)
+K8s-athenz-istio-auth is a controller which converts Athenz domains to Istio RBAC custom resources.
 
 ## Table of Contents
 
 - [Background](#background)
+- [Architecture](#Architecture)
 - [Install](#install)
 - [Configuration](#configuration)
-- [Usage](#usage)
+- [References](#references)
 - [Contribute](#contribute)
 - [License](#license)
 
 ## Background
-We needed a controller that can dynamically fetch Athenz role / policy mappings and convert them to their corresponding 
-Istio custom resources, so we built this controller to allow users to define RBAC through Athenz and have it integrate 
-with the Istio world.
+We needed a controller that can dynamically fetch Athenz role / policy mappings
+and convert them to their corresponding Istio custom resources, so we built this
+controller to allow users to define RBAC through Athenz and have it integrate with
+the Istio world.
 
-The role name in Athenz specifies the target service in the ServiceRole object and its corresponding HTTP methods are 
-listed as policies. The services that can access this target service are members of the role and will be listed in the 
-ServiceRoleBinding.
+### Istio RBAC
```

Comment:
can we include a couple of lines about Athenz similar to the athenz-identity repo: https://github.com/yahoo/k8s-athenz-identity#athenz
> Athenz is an IAM provider which allows users to define role based access control (RBAC) for users and services. It also acts as a certificate authority (CA) by provisioning instances with unique identities through X.509 certificates.